site/cypress/e2e/Cypress-Feature/polls.spec.js

```diff
@@ -349,7 +362,10 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('.correct-box').eq(1).should('not.be.checked');
         cy.get('.poll_response').should('contain', 'Answer 3');
         cy.get('.correct-box').eq(2).should('be.checked');
-        cy.get('textarea').contains('Answer 1').clear().type('Answer 0');
+        cy.get('textarea').contains('Answer 1').then(($el) => {
```

Comment:
Is there a reason to do this one differently than the others?

Reply:
The reason why I changed this to begin with is because cypress recommends against chaining additionally queries/actions after doing something that can modify the state of the page (like `clear()`). The selector `textarea` matches multiple things on the page, and the original code relied on using `contains()` to filter which text area to type `'Answer 0'` in. Since I removed the chaining, I thought this would be concise/readable way to contnue using that specific textarea. I could also just assign an alias to it if you would prefer:
```suggestion
        cy.get('textarea').contains('Answer 1').as('answer1-text');
        cy.get('@answer1-text').clear();
        cy.get('@answer1-text).type('Answer 0');
```

Reply:
I don't really prefer either way, it was just different than the other changes and I was wondering. I think it's okay to leave

Reply:
:+1: